src/tds/codec/token/token_feature_ext_ack.rs

```diff
@@ -7,11 +7,13 @@ pub struct TokenFeatureExtAck {
 }
 
 #[derive(Debug)]
+#[allow(dead_code)]
```

Comment:
All of these are clippy fixes coming from Rust 1.79.0

Reply:
Hm, github's mentioning a bunch of clippy warnings in files that weren't in this PR, I guess those are also to be left until this happens